tests/album-create.test.js

```diff
@@ -0,0 +1,46 @@
+const { expect } = require('chai');
+const request = require('supertest');
+const getDb = require('../src/services/db');
+const app = require('../src/app');
+
+describe('create album', () => {
+  let db;
+  let artist;
+
+  beforeEach(async () => {
+      db = await getDb();
+
+    [artist] = await db.query('SELECT * from Artist');
+
+});
+
+  afterEach(async () => {
+    await db.query('DELETE FROM Albums');
+    await db.close();
+  });
+
+  describe('/album', () => {
+    describe('POST', () => {
+      it('creates a new album in a database', async () => {
+        
+        if(artist[0]) {
```

Comment:
Theres quite a big flaw in this test. The whole thing is being wrapped in a conditional to only run if there is an artist present in the database! You should create an artist as part of the test setup, and remove it as part of the tear down process

Would also be good to add a test to cover trying to add an album to an artist that doesnt exist